MC/bin/o2dpg_sim_workflow.py

```diff
@@ -94,6 +94,11 @@
 
 # MFT reconstruction configuration
 parser.add_argument('--mft-reco-full', action='store_true', help='enables complete mft reco instead of simplified misaligned version')
+parser.add_argument('--mft-assessment-full', action='store_true', help='enables complete assessment of mft reco')
+
+# Global Forward reconstruction configuration
+parser.add_argument('--fwdmatching-assessment-full', action='store_true', help='enables complete assessment of global forward reco')
```

Comment:
Could you please define default values for these arguments? Otherwise, I believe, the value may be `None` and lead to runtime crashes.

Reply:
Hi @sawenzel , Thanks for checking. 
Please note that other parsed parameters are used in boolean comparisons without default values. For instance: `--embedding`, `--combine-smaller-digi`, `--early-tf-cleanup`, `--mft-reco-full`. Default value here does not seem necessary.  Shall I proceed anyway?

Reply:
Indeed. My bad. action=`store_true` does it implicitely.